src/main/java/ch/njol/skript/effects/EffSendTitle.java

```diff
@@ -88,9 +92,23 @@ protected void execute(final Event e) {
 		sub = subtitle != null ? subtitle.getSingle(e) : null;
 		
 		if (TIME_SUPPORTED) {
-			int in = fadeIn != null ? (int) fadeIn.getSingle(e).getTicks_i() : -1,
-			stay = this.stay != null ? (int) this.stay.getSingle(e).getTicks_i() : -1,
-			out = fadeOut != null ? (int) fadeOut.getSingle(e).getTicks_i() : -1;
+			int in, stay, out;
+
+			if (fadeIn != null && fadeIn.getSingle(e) != null) {
+				in = (int) fadeIn.getSingle(e).getTicks_i();
+			} else {
+				in = DEFAULT_FADEIN;
+			}
+			if (this.stay != null && this.stay.getSingle(e) != null) {
+				stay = (int) this.stay.getSingle(e).getTicks_i();
+			} else {
+				stay = DEFAULT_STAY;
+			}
+			if (fadeOut != null && fadeOut.getSingle(e) != null) {
+				out = (int) fadeOut.getSingle(e).getTicks_i();
+			} else {
+				out = DEFAULT_FADEOUT;
+			}
```

Comment:
This will call getSingle twice for 1 execution (for each property), which is unwanted behaviour

Reply:
Fixed